pandas/tests/series/conftest.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+
+import pandas.util.testing as tm
+
+from pandas import Series
+
+
+@pytest.fixture
+def datetime_series():
+    """
+    Fixture for Series of floats with DatetimeIndex
+
+    See pandas.util.testing.makeTimeSeries
```

Comment:
I don't think these See .. comments make sense here, just keep the summary line.

Reply:
Removed those comments